pants.ci.toml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 Pants project contributors.
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+# See https://www.pantsbuild.org/docs/using-pants-in-ci.
+
+[GLOBAL]
+pantsd = false
```

Comment:
why?

Reply:
I think that's common in CI? May not be necessary.

Reply:
Removed this and added `dynamic_ui=False` instead.

Reply:
We originally recomended turning off as Pantsd was getting more stabilization. Then a few weeks ago, I realized we could enable it and that there's some benefit if you do `./pants lint` and `./pants test` as separate runs.

But, last week, we found out users are hitting that Nailgun exception, whose workaround is to disable Pantsd. So, for now, we want Pantsd disabled in CI :/